### DISCUSSION TITLE
Github Actions - Displaying the file when asking for approval

### DISCUSSION THREAD

**Discussion by nathanwong91** [community/community#12682] [Actions]
We've recently started to use Github Actions to deploy resources into AWS while leveraging the `environment` feature so it requires approval from certain groups/people before it goes through. We have it sending that "request" to approve into a slack channel but I'd like to be able to get more information out of it.
Currently when someone goes to approve all they would see is 

![image](https://user-images.githubusercontent.com/59549970/157799442-e27e0a5c-c7d6-420d-a6ec-fff3fe591bc1.png)

going into the job also doesn't give you much information either

<img width="551" alt="image" src="https://user-images.githubusercontent.com/59549970/157799511-f8fe3362-34be-4485-9907-d9ccbf250e05.png">

I've been going through Github API docs using that I can get the workflow_id, environment_id and approve using those but I'm not able to find what the actual file someone wants to deploy.

Is there a way to find what exactly the user is trying to deploy without asking them? There doesn't seem to be an option to show this.

**Comment by ethomson** [comment]
Hi @nathanwong91 - I'll take as feedback that we should investigate providing you some more information about the workflow run - especially things triggered by `pull_request` or `push` where there are code changes. As for your question about getting more insights today - somebody in the community may have some good ideas: https://github.community/c/code-to-cloud/52 is a good resource for Q&A and discussions.
